content.js

```diff
@@ -1,93 +1,88 @@
-
 const myStyleElement = document.createElement("style");
-const cssRuleElement = `span[class*="price"]{
-color: transparent!important;
-text-shadow: rgba(0, 0, 0, 0.8) 10px 0px 10px !important;
-border-style: dotted;
-border-color: #f23838;
-}
+const cssRuleElement = `span[class*="price"],
+span[class="tmm-olp-links"] *
+{
+    color: transparent !important;
+    user-select: none;
+  }
 `;

```

Comment:
```suggestion
const blurr = useBlurr ? "text-shadow: rgba(0, 0, 0, 0.8) -10px 0px 30px !important;" : "";
const cssRuleElement = `span[class*="price"],
span[class="tmm-olp-links"] *
{
    color: transparent !important;
    ${blurr}
    user-select: none;
  }
`;
```

I like this suggestion but lets keep the blur effect as an optional and add it into the readme.